test/script/tskdbdiff.py

```diff
@@ -65,16 +74,15 @@ def run_diff(self):
         """
 
         self._init_diff()
-
         # generate the gold database dumps if necessary     
         if self._generate_gold_dump:       
-            TskDbDiff._dump_output_db_nonbb(self.gold_db_file, self.gold_dump)     
+            TskDbDiff._dump_output_db_nonbb(self.gold_db_file, self.gold_dump, self.isMultiUser, self.pgSettings)     
         if self._generate_gold_bb_dump:        
-            TskDbDiff._dump_output_db_bb(self.gold_db_file, self.gold_bb_dump)
+            TskDbDiff._dump_output_db_bb(self.gold_db_file, self.gold_bb_dump, self.isMultiUser, self.pgSettings)
 
         # generate the output database dumps (both DB and BB)
-        TskDbDiff._dump_output_db_nonbb(self.output_db_file, self._dump)
-        TskDbDiff._dump_output_db_bb(self.output_db_file, self._bb_dump)
+        TskDbDiff._dump_output_db_nonbb(self.output_db_file, self._dump, self.isMultiUser, self.pgSettings)
+        TskDbDiff._dump_output_db_bb(self.output_db_file, self._bb_dump, self.isMultiUser, self.pgSettings)
```

Comment:
Please make a story in JIRA about not doing the dumps twice if _generate_gold_dump is true, thanks! We should be able to just copy the dumps.